helm/values.yaml

```diff
@@ -12,6 +12,13 @@ apiKey: null
 # -- If set, the agent will use the API key in this Secret to authenticate with CloudZero.
 existingSecretName: null
 
+# -- Override the name of the chart. Used in resource naming.
+nameOverride:
```

Comment:
are these now included so that the validation passes? I know they're marked as deprecated. if they're needed here, does it make sense to leave a comment suggesting they not be used? `forceNamespace` especially is not so great, because we haven't been so consistent with using the named template that references it.
really we never should have had that one :/

Reply:
great thanks!